public/index.html

```diff
@@ -25,11 +25,11 @@
   <div id="footer"></div>
   <script crossorigin src="https://ultimate-nike.s3-us-west-1.amazonaws.com/bundle-photo-module.js"></script>
```

Comment:
So since you have your service running now on it's own server you will want to use the IP address followed by /bundle.js rather than relying on S3. It will look like 123.12.12.12:port/bundle.js (stated by Troy)

Reply:
Isn't it better to host the bundle at S3 for more permanence? since my server will be starting/stopping